crates/fuel-core/src/graphql_api/api_service.rs

```diff
@@ -196,11 +197,15 @@ impl RunnableService for GraphqlService {
 
 #[async_trait::async_trait]
 impl RunnableTask for Task {
-    async fn run(&mut self, _: &mut StateWatcher) -> anyhow::Result<bool> {
-        self.server.as_mut().await?;
```

Comment:
This change introduces a non-trivial amount of noise with the removal of the `?` operators.

AFAICT, there isn't a way to implement `Try` for arbitrary types in Rust which is a real bummer.